src/r_data.c

```diff
@@ -848,6 +848,30 @@ void R_InitSpriteLumps(void)
 // killough 4/4/98: Add support for C_START/C_END markers
 //
 
+static byte invul_orig[256];
+
+void R_InvulMode(void)
+{
+  if (colormaps == NULL)
+    return;
+
+  switch (STRICTMODE(invul_mode))
```

Comment:
In strict mode, the invuln sky fix is forced on. It was like that before too, but it seems incorrect (may need to check the logic in `r_plane.c`). Should it be something like this instead?
- At vanilla comp level in strict mode: sky fix is forced off and gray invuln is not allowed.
- At any other comp level in strict mode: sky fix is user preference but gray invuln is still not allowed.

Reply:
> In strict mode, the invuln sky fix is forced on.

Not quite. If you set vanilla complevel and strict mode (don't forget to start a new game), invul sky fix will be forced off.

> At vanilla comp level in strict mode: sky fix is forced off and gray invuln is not allowed.

It works this way already.

> At any other comp level in strict mode: sky fix is user preference but gray invuln is still not allowed.

In DSDA-Doom sky fix is enabled when complevel >= MBF. Technically this can be changed, but `comp_*` settings are even removed from the config, they can only be changed via OPTIONS lump. So, for simplicity's sake, strict mode in Woof just forces the comp settings from DSDA-Doom.